eclass/cuda.eclass

```diff
@@ -81,7 +81,7 @@ cuda_gccdir() {
 
 	if [[ -n ${gcc_bindir} ]]; then
 		if [[ -n ${flag} ]]; then
-			ret="${flag}\"${gcc_bindir}\""
+			ret="${flag}${gcc_bindir}"
```

Comment:
Seems to me that you get a "something"innertext"" kind of thing. At compile time you get a string instead of a path. At any rate the gcc_bindir won't have any spaces in it.
They all are of the type: ${EPREFIX}/usr/*pc-linux-gnu/gcc-bin/${ver}*
I don't really see any problem removing them.